src/constants/MAN_MALE.tsx

```diff
@@ -0,0 +1 @@
+export const MAN_MALE = 'm';
```

Comment:
All constants should be in the same file
Also maybe better to create `enum` for `m` and `f`